src/views/GovernanceReviewTeam/IntakeReview/index.test.tsx

```diff
@@ -41,16 +41,17 @@ describe('The GRT intake review view', () => {
     needsEaSupport: false,
     hasContract: 'The quick brown fox jumps over the lazy dog.',
     grtReviewEmailBody: 'The quick brown fox jumps over the lazy dog.',
-    decidedAt: new Date().toISOString(),
+    decidedAt: new Date(2020, 8, 30).toISOString(),
     businessCaseId: null,
-    submittedAt: new Date().toISOString()
+    submittedAt: new Date(2020, 8, 30).toISOString()
   };
 
   it('renders without crashing', () => {
     shallow(<IntakeReview systemIntake={mockSystemIntake} />);
   });
 
   it('matches the snapshot', () => {
+    Date.now = jest.fn(() => new Date(2020, 8, 30).valueOf());
```

Comment:
Do we need to do any cleanup after this test to reset the value of `Date.now` back to its original value?

Reply:
Good catch! I think we do.

Reply:
@jim I updated the test. Thanks!